sass/govuk_elements.scss

```diff
@@ -18,3 +18,10 @@ $path: "../vendor/govuk_frontend_toolkit/images/";// Path to Frontend Toolkit im
 @import 'govuk-elements-sass/public/sass/elements/_panels';
 @import 'govuk-elements-sass/public/sass/elements/forms/_form-validation';
 @import 'govuk-elements-sass/public/sass/elements/forms/_form-block-labels';
+
+.list.horizontal li {
+  display: list-item;
```

Comment:
Good stuff — slightly surprised GOV.UK Elements didn’t have something like this already.

Do we need `display:list-item` here? It’s the user agent default for `<li>`s, right?